src/main/java/org/jenkinsci/plugins/workflow/libs/AbstractGlobalLibraries.java

```diff
@@ -90,7 +87,15 @@ abstract static class AbstractForJob extends LibraryResolver {
         }
 
         private List<LibraryConfiguration> getLibraries() {
-            return getConfiguration().getLibraries();
+            return getConfiguration()
+                    .getLibraries()
+                    .stream()
+                    .map(this::mayWrapLibrary)
+                    .collect(Collectors.toList());
+        }
+
+        protected LibraryConfiguration mayWrapLibrary(LibraryConfiguration library) {
+            return library;
         }
```

Comment:
If we are even keeping this, it might be clearer to again make it `abstract` so you can compare the trusted vs. untrusted impls directly.